applications/GettingStartedWith/HtmlEditor/index.html

```diff
@@ -0,0 +1,53 @@
+<div id="html-editor">
+    <h2>
+        <img src="/Content/images/doc/21_1/htmlEditor/HtmlEditor.png" alt="HtmlEditor">
+        Formatted Text Editor (HTML Editor)
+    </h2>
+    <br>
+    <p>DevExtreme JavaScript HTML Editor is a client-side WYSIWYG text editor that allows its users to format textual and visual content and store it as HTML or Markdown.</p>
+    <p>Supported features:</p>
+    <ul>
+        <li>Inline formats:
+            <ul>
+                <li><strong>Bold</strong>, <em>italic</em>, <s>strikethrough</s> text formatting</li>
+                <li>Font, size, color changes (HTML only)</li>
+            </ul>
+        </li>
+        <li>Block formats:
+            <ul>
+                <li>Headers</li>
+                <li>Text alignment</li>
```

Comment:
This item seems out of place. (I understand that technically it's probably something you apply to a "block" and that's why you list it here.) It seems like I can turn my text into different things - a header, a list, a code block, a quote... But I can change text alignment in all those things - it's not another type of thing. Maybe it's better to list this item separately.  I stumble on it every time I read this list.